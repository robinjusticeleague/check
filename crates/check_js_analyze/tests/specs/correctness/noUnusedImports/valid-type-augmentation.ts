/* should not generate diagnostics */

// See: https://github.com/checkjs/check/issues/6669

import type {} from "@mui/lab/themeAugmentation";
