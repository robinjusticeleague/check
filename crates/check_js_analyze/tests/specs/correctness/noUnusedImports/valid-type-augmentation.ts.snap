---
source: crates/check_js_analyze/tests/spec_tests.rs
expression: valid-type-augmentation.ts
---
# Input
```ts
/* should not generate diagnostics */

// See: https://github.com/checkjs/check/issues/6669

import type {} from "@mui/lab/themeAugmentation";

```
